src/jit/treelifeupdater.cpp

```diff
@@ -245,6 +245,13 @@ void TreeLifeUpdater<ForCodeGen>::UpdateLifeVar(GenTree* tree)
                 }
 #endif // DEBUG
             }
+
+#ifdef USING_VARIABLE_LIVE_RANGE
+            // For each of the LclVarDsc that are reporting change, variable or fields
+            VariableLiveKeeper* varLiveKeeper = compiler->getVariableLiveKeeper();
+            varLiveKeeper->siStartOrCloseVariableLiveRanges(&varDeltaSet, isBorn, isDying);
```

Comment:
I agree that it's verbose. We can do this:
> ```
> compiler->siStartOrCloseVariableLiveRanges(&varDeltaSet, isBorn, isDying);
> ```
But the intention of having just a getter was to avoid adding more methods to Compiler.

I preffer to do the other way:
> ```
> compiler->getVariableLiveKeeper()->siStartOrCloseVariableLiveRanges(&varDeltaSet, isBorn, isDying);
> ```

and adding the flag inside the function
> Also, maybe the `#ifdef USING_VARIABLE_LIVE_RANGE` could be put in the function itself, to avoid putting it at every call site?

The problem with that is if someone adds/moves a call to this method and the flag "USING_VARIABLE_LIVE_RANGE" is disable, it will compile without errors. Is that the expected behavior on changes?